dist/Makefile

```diff
@@ -1,15 +1,15 @@
 HOSTPORT = 12345
 IMAGENAME = "malibu"
 
-.PHONY: docker run-docker
+.PHONY: docker docker-run docker-run-%
 
 docker:
 	docker build -t $(IMAGENAME) ..
 
-run-docker: docker
+docker-run: docker
 	@echo "Running 'malibu' container on port $(HOSTPORT)"
-	@echo "Open 'http://localhost:$(HOSTPORT)/isbn/suche.html' to access it"
-	docker run --rm -it -p $(HOSTPORT):80 "$(IMAGENAME)"
+	@echo "Open 'http://localhost:$(HOSTPORT)/malibu/isbn/suche.html' to access it"
```

Comment:
I don't know exactly. But maybe just add another line of explanation for windows for the moment, e.g. something like
```
@echo "Running 'malibu' container on port $(HOSTPORT)"
@echo "Open 'http://localhost:$(HOSTPORT)/malibu/isbn/suche.html' to access it"
@echo "Alternatively substitute 'localhost' with the docker-machine ip"
```